indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -56,3 +67,55 @@ def _checkParams(self, params):
 
     def _process(self):
         return self.abstract_file.send()
+
+
+class RHDisplayReviewableTracks(RHAbstractsBase):
+    def _checkProtection(self):
+        RHAbstractsBase._checkProtection(self)
+        if not session.user:
+            raise Forbidden
+
+    def _process(self):
+        query = Track.query.with_parent(self.event_new)
+        track_reviewer_abstract_count = get_track_reviewer_abstract_counts(self.event_new, session.user)
+
+        # if the user is not a global convener only show their tracks
+        if (session.user not in self.event_new.global_abstract_reviewers and
+                session.user not in self.event_new.global_conveners):
+            query = query.filter(db.or_(Track.conveners.any(User.id == session.user.id),
+                                        Track.abstract_reviewers.any(User.id == session.user.id)))
+
+        return WPDisplayAbstractsReviewing.render_template('display/tracks.html', self._conf, event=self.event_new,
+                                                           tracks=query.all(),
+                                                           abstract_count=track_reviewer_abstract_count)
```

Comment:
It would look a bit nicer if this line was swapped with the line above, but that's just something completely OCD so you can ignore it if you want!